tox.ini

```diff
@@ -0,0 +1,13 @@
+[tox]
+skipsdist = true
+envlist = tests, flake8
```

Comment:
+1 - I volunteer to help with a sweep of our projects, at some point in the future, given priorities + time :-)